### ISSUE TITLE
Compatibility with different pytorch-based models

### ISSUE THREAD

**Issue by cassini-fly** [ActiveIntelligentSystemsLab/pytorch_enet_ros#5]
Hi,

Thanks for these great stuff. Is this package compatible with different models as well? (of course after doing the .pth to .pt conversions)
As a follow-up question, could you specify where in the code has to be changed so that above can be achieved?

**Comment by ShigemichiMatsuzaki** [comment]
Hi,

Thank you for your interest and sorry for the late reply.

The nodes in this package assume a specific network structure and are not compatible with others as it is.
(It's not ENet anymore. Sorry for the confusion)
Specifically, for example [pytorch_cpp_wrapper_seg_trav.cpp](https://github.com/ActiveIntelligentSystemsLab/pytorch_enet_ros/blob/master/src/pytorch_cpp_wrapper_seg_trav.cpp) assume that the network has two segmentation branches (main: 0, and aux: 1), and another branch for estimating pixel-wise probability value.
When you use this program, your network must have this specific output interface.

I suggest doing the following.
1. Using `PyTorchCppWrapperBase` as a superclass, write a new wrapper class for your network.
Basically, you can just slightly modify the function `get_output()` in  [pytorch_cpp_wrapper_seg_trav.cpp](https://github.com/ActiveIntelligentSystemsLab/pytorch_enet_ros/blob/master/src/pytorch_cpp_wrapper_seg_trav.cpp) match your network.
2. Write a new code for a ROS node.
Following [pytorch_cpp_wrapper_seg_trav.cpp](https://github.com/ActiveIntelligentSystemsLab/pytorch_enet_ros/blob/master/src/pytorch_enet_ros.cpp), you can modify `inference()` and some other parts accordingly.
